root.tf

```diff
@@ -315,3 +326,11 @@ module "ecr_image_scan_notification_lambda" {
   project                       = "tdr"
   lambda_ecr_scan_notifications = true
 }
+
+module "jenkins_backup_maintenance_window" {
+  source = "./tdr-terraform-modules/ssm_maintenance_window"
+  command = "docker exec $(docker ps -aq -f ancestor=${data.aws_ssm_parameter.mgmt_account_number.value}.dkr.ecr.eu-west-2.amazonaws.com/jenkins) /opt/backup.sh"
+  instance_name = "jenkins-task-definition-mgmt"
+  name = "tdr-jenkins-backup-window"
+  schedule = "cron(0 0 18 ? * MON-FRI *)"
+}
```

Comment:
Missing line break

Reply:
I've added it in. And I didn't run fmt :cry: I did mean to though.

Reply:
What do you both think about putting [TDR-163](https://national-archives.atlassian.net/browse/TDR-163) (running `terraform fmt -check` in Jenkins) into the next sprint? Seems like it would save time.